Pipfile

```diff
@@ -9,5 +9,10 @@ requests = "~=2.32.3"
 jira = "~=3.0.0"
 logging-formatter-anticrlf = "~=1.2.1"
 
+[dev-packages]
+bandit = "~=1.7.9"
+black = "~=24.4.2"
+flake8 = "~=7.1.0"
+
 [requires]
 python_version = "3"
```

Comment:
Nitpick: Maybe we can be more specific on the version required given I don't see we have tests for all python 3.x.

Reply:
Unfortunately this field doesn't support a version range, and specifying a particular 3.x version here requires the user to use precisely that version. I think that would break existing users (as they may be using a variety of different supported versions), and would make this brittle when e.g. GitHub hosted Actions runners upgrade the default version of Python.